tests/pos/nullAsInstanceOf.scala

```diff
@@ -1,11 +0,0 @@
-object Test {
```

Comment:
Do we want to do this for new tests to? I thought `LegacyApp` was only a compatiblity hack.